browser/js/common/directives/cart/cartItem.html

```diff
@@ -0,0 +1,31 @@
+
+<div id="cart_item" class="container">
+	<div class="row">
```

Comment:
I don't think you need a row here; there are no other items and it's housed in a div already.